src/utils.js

```diff
@@ -238,6 +240,22 @@ function urlValidation(urlString) {
 
 const normalizeFileUpload = (value) => value && value.split(/\r\n|\n/);
 
+function convertChannelCodeToDisplayName(code) {
+  if (code === 'SAP') {
+    return code;
+  }
+  if (code === 'DEGREED2') {
+    return 'Degreed';
+  }
+  if (code === 'DEGREED') {
+    return 'Degreed_old';
```

Comment:
Degreed_old? Maybe we can come up with a better name - 'Degreed v1'? 'Degreed (Deprecated)'?

Reply:
I like (Deprecated) just to be uber clear